plugins/GSdx/GSHwHack.cpp

```diff
@@ -762,10 +763,10 @@ bool GSC_Kunoichi(const GSFrameInfo& fi, int& skip)
 		{
 			skip = 1;
 		}
-		if(fi.TME)
+		// Removes burning air effect, the effect causes major slowdowns.
```

Comment:
Put this above the skip value.

 			{
 				// Removes burning air effect, the effect causes major slowdowns.
 				skip = 1;
 			}